rbac/internal/views.py

```diff
@@ -1720,6 +1721,35 @@ def check_inventory(request):
         )
 
 
+def check_bootstrapped_tenants(request):
+    """POST to check if bootstrapped tenant is correct on relations api."""
+    mappings = []
+    relations_bootstrapped_tenants = []
+    # Get all tenants with mappings i.e are bootstrapped
+    bootstrapped_tenants = Tenant.objects.filter(tenant_mapping__isnull=False)
```

Comment:
can this endpoint be just per tenant ? I think this will be timeout if go through all tenants